core/src/main/resources/error/error-classes.json

```diff
@@ -641,6 +641,12 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_DATA_TYPE_SCHEMA" : {
```

Comment:
We already have the error class `INVALID_SCHEMA`. Can you re-use it or make it more generic? cc @srielau